src/game/text.c

```diff
@@ -56,6 +56,14 @@ static int8_t m_TextASCIIMap[95] = {
 
 static void Text_DrawText(TEXTSTRING *textstring);
 
+void Text_ReplaceChar(char *str, char orig_char, char rep_char)
```

Comment:
I don't think this should be put here. This module is about displaying TEXTSTRING not manipulation of `char*`. I'd move this to `game/shell.c` and inline this logic under a new function, `GetScreenshotName`.